css/style.css

```diff
@@ -0,0 +1,903 @@
+  *,
+*::after,
+*::before {

```

Comment:
Looks very strange... why do you need to use this selector with pseudo element?

Reply:
Thanks
reset took from old works
Fixed it
```suggestion 
* {
      box-sizing: border-box;
  }
```